.github/workflows/gutenberg-packages-update.yml

```
@@ -25,18 +25,18 @@ jobs:
         id: last-release
         run: |
           PR_TITLE=$(gh api -X GET search/issues -f q='${{ env.QUERY }}' -f sort='created' -f order='desc' --jq '.items.[].title' | head -n 1)
-          LAST_VERSION=$(node -e "'$PR_TITLE' && console.log( '$PR_TITLE'.match('v(.+) release')[1] )")
+          LAST_VERSION=$(sed -r 's/.+ v(.+) .+/\1/' <<< "$PR_TITLE")
           echo "::set-output name=version::$(echo "$LAST_VERSION")"
```

In order to ensure only valid version numbers are used, this could could be checked as follows:

```suggestion
          if egrep -q '^[0-9][0-9]*(\.[0-9][0-9]*)*$' <<< "$LAST_VERSION"; then
            echo "::set-output name=version::$(echo "$LAST_VERSION")"
          fi
```

---

And maybe `exit 1` if `else`?

---

Maybe not a non-zero exit, as that would mark the job as a failure which is techincally not true... unless that's what we want?

---

At least the job should be skipped entirely.

But at the moment there are no such merged PRs in the repo: https://github.com/ampproject/amp-wp/pulls?q=is%3Apr+author%3Aapp%2Fgithub-actions+is%3Aopen+is%3Amerged+in%3Atitle+Update+Gutenberg+packages+after

I guess in that case `LAST_VER` will be empty? So then maybe it should be set to `0.0.0`:

```suggestion
          if ! egrep -q '^[0-9][0-9]*(\.[0-9][0-9]*)*$' <<< "$LAST_VERSION"; then
            LAST_VERSION='0.0.0'
          fi
          echo "::set-output name=version::$(echo "$LAST_VERSION")"
```

---

That works 👍.